File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    
    T = len(t)
    P = len(p)
    num_p = int(p)
    
    for i in range(T-P+1):
        num = ""
        
        for j in range(i, i+P):
            num += t[j]
            
        if int(num) <= num_p:
            answer += 1
    
    return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    var answer = 0;
    
    const lenT = t.length
    const lenP = p.length
    
    for (let i=0; i < (lenT - lenP + 1); i++) {
        let temp = ""
        
        for (let j=0; j < lenP; j++ ) {
            temp += t[i+j]
        }
        
        if (Number(temp) <= Number(p) ) {
            answer++
        }
    }
    
    return answer;
}